packages/snapshot-listener/src/utils/fetchSnapshot.ts

```diff
@@ -0,0 +1,60 @@
+import fetch from "node-fetch";
+import dotenv from "dotenv";
+dotenv.config();
+
+export default async function fetchNewSnapshotProposal(id: string) {
+  try {
+    const environment = process.env.NODE_ENV || "development";
+
+    if (environment === "development") {
+      const dummyProposal = {
+        data: {
+          proposal: {
+            id: "proposal/0x1234",
+            title: "Dummy Proposal",
+            body: "This is a dummy proposal",
+            choices: ["Yes", "No"],
+            space: "dummy-space.eth",
+            start: 1660680000, // TODO: is this the format we get back?
+            end: 1670680000,
+          },
+        },
+      };
+
+      return dummyProposal;
+    }
+
+    const response = await fetch("https://hub.snapshot.org/graphql", {
```

Comment:
Is this code replicated across snapshot-listener / commonwealth repos? If so, we should move to common-common.

Reply:
the implementation is a bit different in commonwealth, there it requires DB instance as it does a check against the DB for existing data